README.md

```diff
@@ -1,104 +1,216 @@
-  -b          path to BF/CBF file [required]
-  -o          path to output TSV file [required]
-  --distinct
+options:
+  -h, --help  show this help message and exit
+  -b B        path to BF/CBF file
+  -s S        path to spaced seeds file
+  -o O        path to output TSV file
 ```
 
-## `rarity`
-
-Calculate TF-IDF information from sequencing data. Two counting Bloom filters will be stored. One for *k*-mer/seed counts (TF) and other for depths (DF). The number of reads ($N$) will also be printed to stdout. To compute the TF-IDF of an element with $h$ as its hash array, use this formula:
+- The input data can be any FASTA, FASTQ, or BAM. Compressed files are supported. To read from stdin, use `-`.
+- If the data structure contains spaced seed information, pass the path to the spaced seeds text file to `-s`.
 
-$$ \text{TF-IDF}(h) = TF(h) * log \dfrac{N + 1}{DF(h) + 1} $$
 
-Note that the $+1$ in the $log$ operation is to prevent division by zero.
+## `rarity`
 
-### Usage
+Calculate TF-IDF information from sequencing data. This module outputs two counting Bloom filters: One for *k*-mer/seed counts (TF) and other for depths (DF). The number of reads will also be printed to stdout.
 
 ```
-rarity [--help] [--version] [-k VAR] [-s VAR] -f VAR [-e VAR] [-b VAR] [--long] [-t VAR] -o VAR reads
+Usage: rarity [--help] [--version] [-k VAR] [-s VAR] -f VAR [-e VAR] [-b VAR] [--long] [-t VAR] -o VAR reads
 
 Positional arguments:
   reads          path to sequencing data file(s) [nargs: 0 or more] [required]
 
 Optional arguments:
-  -h, --help     shows help message and exits
-  -v, --version  prints version information and exits
-  -k             k-mer length
-  -s             path to spaced seeds file (one per line, if -k not specified)
+  -h, --help     shows help message and exits 
+  -v, --version  prints version information and exits 
+  -k             k-mer length 
+  -s             path to spaced seeds file (one per line, if -k not specified) 
   -f             path to k-mer spectrum file (from ntCard) [required]
   -e             target output false positive rate [nargs=0..1] [default: 0.0001]
-  -b             output CBF size (bytes)
-  --long         optimize for long read data
+  -b             output BF/CBF size (bytes) 
+  --long         optimize for long read data 
   -t             number of threads [nargs=0..1] [default: 1]
-  -o             prefix for storing output counting bloom filters [required]
+  -o             path to store output file [required]
 ```
 
+- Refer to the `filter` module's notes for information about the parameters.
+
 ## `hist`
 
-### Usage
+Fits a mixture model to the *k*-mer count histogram to estimate *k*-mer error and heterozygous probabilities, genome size, and thresholds useful for the `filter` command.
 
 ```
-hist [--help] [--version] histogram
-
-Positional arguments:
-  histogram      path to k-mer spectrum file (from ntCard) [required]
+usage: main.py [-h]
```

Comment:
Should be `ntStat hist` I would think?

Reply:
Fixed in 2057c6844555100d86789b492ba3d2a316aa628a